manage.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python3
+# Management script for dwyl smart home
+
+import argparse
+import subprocess
+import os
+import sys
+import shlex
+
+from contextlib import contextmanager
+
+# Global verbose flag to toggle on/off verbose output
+VERBOSE=False
+
+# Declare global constants
+HUB_SERVER_REPO="https://github.com/dwyl/smart-home-auth-server.git"
+FIRMWARE_REPO="https://github.com/dwyl/smart-home-firmware.git"
+
+# Declare the start message here so we don't clog up the rest of the code
+START_MESSAGE="""
+Smart home is now setup for development.
+
+To setup your development evironment:
+    source .env
+
+To run the hub server:
+
+    cd smart-home-auth-server
+    mix phx.server
+
+To run the firmware development build:
+    cd smart-home-firmware
+    iex -S mix
+
+If these error, please run:
+    ./manage.py clean
+    ./manage.py --verbose install
+and file and issue with the output.
+"""
+
+# Emulate a shells `cd`
+# https://stackoverflow.com/questions/431684/
+@contextmanager
+def cd(newdir):
+    prevdir = os.getcwd()
+    os.chdir(os.path.expanduser(newdir))
+    try:
+        yield
+    finally:
+        os.chdir(prevdir)
+
+# Helper function to allow for same line I/O without repeating ourselves
+def out(line):
+  print(line, end="", flush=True)
+
+# Run a command, and display the result based on wether VERBOSE is set or not
+def run(*args):
+  return subprocess.run(args, capture_output=not VERBOSE)
+
+def write_env(key):
+  with open(".env", "w") as f:
```

Comment:
@th0mas I'm re-reading the script to confirm my understanding before running it.
Am I correct in thinking that this `write_env/1` will ***overwrite*** the contents of the `.env` file?
I ask because if there are any _other_ environment variables e.g. `DATBASE_URL` or `PORT`, etc. they will be lost in this write.

I wrote a similar (_but more verbose_) function in `write_env/2`: https://github.com/dwyl/auth/blob/7abbd6f69e183e368cb0f49fd22047b67e2d89f0/priv/repo/seeds.exs#L61
That preserves the _existing_ entries in the `.env` file and simply overwrites the desired `key, value` pair.
Could be worth considering as an enhancement so we can have more environment variables. 💭

Reply:
Yeah, whoops, that's definitely a bug! 
I'll look into and change that now

Reply:
Cool. thanks for updating it. 👍